tests/test_dynamic_groups.c

```diff
@@ -7,6 +7,18 @@
 #include "rtrlib/rtr_mgr_private.h"
 #include "rtrlib/spki/hashtable/tommyds-1.8/tommylist.h"
 
+const int connection_timeout = 20;
+enum rtr_mgr_status connection_status = -1;
+
+static void connection_status_callback(const struct rtr_mgr_group *group,
+				       enum rtr_mgr_status status,
+				       const struct rtr_socket *socket,
+				       void *data)
+{
+	if (status == RTR_MGR_ERROR)
```

Comment:
is this necessary? Why not just update the status all the time, the check for error is done in the loop below anyway.